server.js

```diff
@@ -21,8 +21,8 @@
 const http = require('http');

```

Comment:
Consider using a more descriptive name for the PORT constant, such as HTTP_PORT, to improve code readability.